evaluator.py

```diff
@@ -137,13 +137,22 @@ def final(**available_fileds):
     return middle
 
 @default
-@inputs(COSTS)
-def trading_costs(costs):
+@inputs(PNLS)
+def max_drawdown(pnls):
     """
-    Average annual costs of trading
+    Max drawdown in simualtion period.
+    If the drawdown if positive, it will return 0.
     """
-    aggregated_costs = sum(np.array(list(costs.values())))
-    return np.mean(aggregated_costs) * 252
+    aggregated_pnls = sum(np.array(list(pnls.values())))
+    accumulated_pnls = [0.]
+    for pnl in aggregated_pnls:
+        accumulated_pnls.append(accumulated_pnls[-1] + pnl)
```

Comment:
Can you try to use numpy built-in functions as much as possible to optimize the performance?
For example, you can use [the builtin acculmulate function](https://numpy.org/doc/stable/reference/generated/numpy.ufunc.accumulate.html).